daffodil-core/src/main/java/org/apache/daffodil/api/debugger/package-info.java

```diff
@@ -20,27 +20,21 @@
  *
  * <h2>Overview</h2>
  * <p>
- * Daffodil comes with one prebuilt debugger, the {@link org.apache.daffodil.api.debugger.TraceDebuggerRunner}, which outputs
+ * Daffodil comes with one prebuilt debugger, the TraceDebugger which is a 
+ * {@link org.apache.daffodil.api.debugger.Debugger}, which outputs
  * verbose information during the parsing processes, which can be used to aid
  * in debugging a DFDL schema. For example, the {@link
- * org.apache.daffodil.api.debugger.TraceDebuggerRunner} can be used like so:
+ * org.apache.daffodil.api.debugger.Debugger} TraceDebugger can be used like so:
```

Comment:
change this to "trace debugger" to make it clear that a `TraceDebugger` isn't an actual class/interface.